docs/overview/about.md

```diff
@@ -0,0 +1,80 @@
+# About Vald <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+
+- [What is Vald?](#what-is-vald)
+  - [What Vald can do?](#what-vald-can-do)
+  - [Use cases](#use-cases)
+- [Why Vald?](#why-vald)
+- [How does Vald work?](#how-does-vald-work)
+- [Try Vald](#try-vald)
+
+### What is Vald?
+
+<!-- copied -->
+Vald is a highly scalable distributed fast approximate nearest neighbor dense vector search engine.
+Vald is designed and implemented based on Cloud-Native architecture.
+It uses the fastest ANN Algorithm NGT to search neighbors.
+Vald has automatic vector indexing and index backup, and horizontal scaling which made for searching from billions of feature vector data.
+Vald is easy to use, feature-rich and highly customizable as you needed.
+
+#### What Vald can do?
+
+<!--  copied -->
+
+- Asynchronize Auto Indexing
+    - Usually the graph requires locking during indexing, which cause stop-the-world. But Vald uses distributed index graph so it continues to work during indexing.
+
+- Customizable Ingress/Egress Filtering
+    - Vald implements it's own highly customizable Ingress/Egress filter.
+    - Which can be configured to fit the gRPC interface.
+        - Ingress Filter: Ability to Vectorize through filter on request.
+        - Egress Filter: rerank or filter the searching result with your own algorithm.
+
+- Cloud-native based vector searching engine
+    - Horizontal scalable on memory and cpu for your demand.
+
+- Auto Indexing Backup
+    - Vald has auto index backup feature using MySQL + Redis or Cassandra which enables disaster recovery.
+
+- Distributed Indexing
+    - Vald distribute vector index to multiple agent, each agent stores different index.
+
+- Index Replication
+    - Vald stores each index in multiple agents which enables index replicas.
+    - Automatically rebalance the replica when some Vald agent goes down.
+
+- Easy to use
+    - Vald can be easily installed in a few steps.
+
+- Highly customizable
+    - You can configure the number of vector dimension, the number of replica and etc.
+
+- Multi language supported
+    - Golang, Java, Nodejs and python is supported.
+
+#### Use cases
+
+Vald support similarity searching.
+
+- Related image search
+- Speech recognition
+- Everything you can vectorize :)
```

Comment:
Could you please mean the following word ? 🙏 
```
:)
```

Reply:
it has no meaning XD

Reply:
ok